Build/Scripts/CompileAppInBcContainer.ps1

```diff
@@ -20,15 +24,19 @@ if($app)
     if($appBuildMode -eq 'Default') {
         $parameters["Features"] += @("lcgtranslationfile")
     }
+
+    # Restore the baseline app and generate the AppSourceCop.json file
+    if (($parameters.ContainsKey("EnableAppSourceCop") -and $parameters["EnableAppSourceCop"]) -or ($parameters.ContainsKey("EnablePerTenantExtensionCop") -and $parameters["EnablePerTenantExtensionCop"])) {
+        Set-BreakingChangesCheck -AppSymbolsFolder $parameters["appSymbolsFolder"] -AppProjectFolder $parameters["appProjectFolder"] -BuildMode $appBuildMode | Out-Null
```

Comment:
What about `Enable-BreakingChangesCheck`?

Don't have a strong preference, just think _enable_ sounds better in the context.